cnames_active.js

```diff
@@ -47,6 +47,7 @@ var cnames_active = {
   "1c": "oknosoft.github.io/1c",
   "1diaboliko84": "1diaboliko84.github.io",
   "1ib": "1ib.github.io",
+  "2176608":"cname.vercel-dns.com",
```

Comment:
```suggestion
  "2176608": "cname.vercel-dns.com", // noCF
```
As the CI error indicates:
- You are missing a space between the key and the value in your entry
- When using Vercel, the Cloudflare proxy should be disabled with the `// noCF` comment